lib/model.js

```diff
@@ -2590,7 +2590,7 @@ class Model {
    * @param  {Boolean}      [options.sideEffects=true] Whether or not to update the side effects of any virtual setters.
    * @param  {Boolean}      [options.individualHooks=false] Run before / after update hooks?. If true, this will execute a SELECT followed by individual UPDATEs. A select is needed, because the row data needs to be passed to the hooks
    * @param  {Boolean}      [options.returning=false]       Return the affected rows (only for postgres)
-   * @param  {Number}       [options.limit]                 How many rows to update (only for mysql and mariadb, implemented as TOP(n) for MSSQL)
+   * @param  {Number}       [options.limit]                 How many rows to update (only for mysql and mariadb, implemented as TOP(n) for MSSQL; for sqlite it is supported only when rowid is present)
```

Comment:
isnt rowId always available?

Reply:
It is enabled by default, but can be absent if created with `WITHOUT ROWID`. Ref: https://sqlite.org/withoutrowid.html